gpio/README.md

```diff
@@ -65,7 +73,7 @@ Additionally the Sensor can detect toggle events and report the time of the even
 ### Dependencies
 
 The user running sensor_reporter must have permission to access the GPIO pins.
-To grant the `sensorReporter` user GPIO permissions add the user to the group `gpio`:  `sudo adduser sensorReporter gpio`
+To grant the `sensorReporter` user GPIO permissions add the user to the group `gpio`:  `$ sudo adduser sensorReporter gpio`
```

Comment:
I never added the $ to commands like this to make it easier to copy and paste. I've seen it done either way in other repos. Maybe it makes sense to make it a separate code block which I think will a copy icon.

Reply:
We should be consistent so I'd say yes.